Tools/Verification/DuplicatiVerify.py

```diff
@@ -1,98 +1,102 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 
 """
-This file is a standalone python script that is tested with python 2.7
+This file is a standalone python script that is tested with python 2.7 & 
+python 3.8
 If Duplicati is producing a backup with the option --upload-verification-file,
 it will produce a *-verification.json file on the backend, which can be verified
 by this script. Simply run this script with the path to the backup
 folder as input, and it will verify all *-verification.json files in the folder.
 """
 
+from __future__ import print_function
 import sys
 import os
 import string
 import json
 import base64
 import codecs
 from hashlib import sha256
+import argparse
 
 """ Utility function to return byte chunks from a binary file """
-def bytes_from_file(filename, chunksize=8192):
+def bytes_from_file(filename, chunksize=8192*1024):
     with open(filename, "rb") as f:
         while True:
             chunk = f.read(chunksize)
             if chunk:
-                for b in chunk:
-                    yield b
+                #for b in chunk:
+                #    yield b
+                yield chunk

```

Comment:
What was the purpose of changing the default `chunksize` and removing the `for` loop?